openstack/sharedfilesystems/v2/shares/requests.go

```diff
@@ -267,3 +267,77 @@ func ListAccessRights(client *gophercloud.ServiceClient, id string) (r ListAcces
 	})
 	return
 }
+
+// ExtendOptsBuilder allows extensions to add additional parameters to the
+// Extend request.
+type ExtendOptsBuilder interface {
+	ToExtendMap() (map[string]interface{}, error)
+}
+
+// ExtendOpts contains the options for creation of a Extend request.
```

Comment:
How about:

ExtendOpts contains options for extending a Share.

Reply:
done